package.py

```diff
@@ -1,6 +1,6 @@
 name = "kitsu"
 title = "Kitsu"
-version = "1.2.4-dev.1"
+version = "1.2.4-appstart.19"
```

Comment:
Please, Don't modify the version. 
```suggestion
version = "1.2.4-dev.1"
```